pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -155,21 +160,32 @@ public PinotHelixResourceManager(@Nonnull String zkURL, @Nonnull String helixClu
     _isSingleTenantCluster = isSingleTenantCluster;
     _enableBatchMessageMode = enableBatchMessageMode;
     _allowHLCTables = allowHLCTables;
+    _helixInstanceType = helixInstanceType;
+
+    _helixZkManager =
```

Comment:
Set up HelixManager and LeadControllerManager in the start()

Reply:
Pass HelixManager as param in start() method.

Reply:
Done.